### PR TITLE
Update ProductUpdater and pim_catalog.updater.product

### PR DESCRIPTION
**Pull request by userz58**
to add new fields to the product (not attributes), only updating the service

### CODE REVIEW

**Comment by juliensnz** [comment]
Hello @userz58,

Thanks for this nice contribution. This is GTM for me and we will try to merge it asap.

Regards

**Comment by willy-ahva** [comment]
GTM

**Comment by SamirBoulil** [comment]
GTM

File: src/Pim/Component/Catalog/Updater/ProductUpdater.php [hunk 1]
```diff
@@ -25,7 +25,9 @@ class ProductUpdater implements ObjectUpdaterInterface
 
     /** @var ProductTemplateUpdaterInterface */
     protected $templateUpdater;
-
+    
+    protected $supportedFields = [];
```
Comment (phaseinducer) [inline src/Pim/Component/Catalog/Updater/ProductUpdater.php]: Please add `/** @var array */` here

**Comment by nidup** [comment]
Hi @userz58 
Thank you for your contribution!
To be able to merge your Pull Request we need that you read and sign our contributor agreement http://www.akeneo.com/contributor-license-agreement/
May you also fix the phpspec (unit test)?
Don't hesitate if you need help or any further information.

**Comment by nidup** [comment]
thx for the PR, merged (i fixed php specs on master)

**Pull request merged**